models/bear.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const Joi = require('joi');
+
+//Schema for the bear
+const bearSchema = new mongoose.Schema({
+  name: String,
```

Comment:
Are any of the columns required? You may want to specify that in the Schema

Reply:
ok Boss...
i will consider that